packages/firebase_messaging/firebase_messaging/android/src/main/java/io/flutter/plugins/firebase/messaging/FlutterFirebaseMessagingBackgroundExecutor.java

```diff
@@ -227,29 +228,38 @@ public void notImplemented() {
             }
           };
     }
+    // RemoteMessage is passed as byte array. Check it exists first
+    byte[] parcelBytes =
+        intent.getByteArrayExtra(FlutterFirebaseMessagingUtils.EXTRA_REMOTE_MESSAGE);
+    if (parcelBytes != null) {
+      Parcel parcel = Parcel.obtain();
```

Comment:
This cannot fail? Or it should be part of the try/catch?

Reply:
It should never fail.